stdlib/array.gr

```diff
@@ -728,6 +805,9 @@ provide let filter = (fn, array) => {
  * @param fn: The function to call on each element, where the returned value indicates if the element satisfies the condition
  * @param array: The array to iterate
  * @returns The new array containing elements where `fn` returned `true`
+ * 
+ * @example
+ * Array.filteri((a, i) => a < 9 && i > 0, [> 5, 6, 7, 8, 9]) // [> 6, 7, 8]
```

Comment:
Same As Above:
```suggestion
 * Array.filteri((a, i) => a < 9 && i > 0, [> 5, 6, 7, 8, 9]) == [> 6, 7, 8]
```

Reply:
Same comment as before. This is not returning void so simply in line the result.....